packages/vue/src/DataDisplay/ListItem/components/OcPayment.vue

```diff
@@ -60,7 +60,17 @@ onBeforeUnmount(() => {
     <div class="flex items-center gap-x-5">
       <div class="flex-1 flex items-center gap-x-3 font-medium capitalize">
         <slot name="logo" />
-        {{ title }}
+        <div class="flex items-center gap-x-3 overflow-hidden">
```

Comment:
hi @mykhailo-diakovych  plase update all props chipProps OcPayment in dashboard

Reply:
Hi @kewcoder  in dashboard we don't have any list item with type "payment" with chipProps